ui/src/setupProxy.js

```diff
@@ -11,9 +12,19 @@ module.exports = function(app) {
     path: '/',
     httpOnly: true
   }
+
```

Comment:
Let's add a comment here that this line is here to allow access to parsed request body.

Reply:
done